docs/04-scaling-cluster.md

```diff
@@ -46,27 +47,24 @@ a little longer.
 ### Note
 You can alter the `Machine Set` count in several ways in the web UI. You can
 also perform the same operation via the CLI by using the `oc edit` command on
-the `machineset` in the `openshift-cluster-api` project.
-
-# WARNING
-**WARNING** the autoscaling example is not complete.
+the `machineset` in the `openshift-machine-api` project.
 
 ## Automatic Cluster Scale Up
 Automatic scale based on workload is possible provided there is a
 configuration specified to do so. From the command line, take a look at the
 `machinesets` (they will look the same as in the web UI):
 
-    oc get machinesets -n openshift-cluster-api
+    oc get machinesets -n openshift-machine-api
 
 You will see something like (differing depending on AWS region and `clusterid`):
 
-    NAME                       DESIRED   CURRENT   READY     AGE
-    190104-worker-us-east-1a   1         1         1         4h
-    190104-worker-us-east-1b   1         1         1         4h
-    190104-worker-us-east-1c   1         1         1         4h
-    190104-worker-us-east-1d   0         0                   4h
-    190104-worker-us-east-1e   0         0                   4h
-    190104-worker-us-east-1f   0         0                   4h
+    NAME                                    DESIRED   CURRENT   READY   AVAILABLE   AGE
+    beta-190227-1-7fj4t-worker-us-east-1a   1         1         1       1           64m
```

Comment:
We'd better make sure all examples cluster name are the same.

Reply:
Why?

Reply:
The current content is correct, just to make the doc demos the examples with the same cluster, looks consistent, no ambiguity.